docs/components/responsive-image.mdx

```diff
@@ -0,0 +1,78 @@
+---
+id: responsive-image
+title: ResponsiveImage
+---
+
+## `ResponsiveImage`
+
+If you need an image that needs to be a **[responsive image](https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images)** and you want to a react component to generate the `srcset` for this image you can use the `ResponsiveImage` component.
+
+Its props are as follows:
+
+| Prop            | Type    | Description                                                                   |
+|-----------------|---------|-------------------------------------------------------------------------------|
+|src              | string  | This is the src for the image                                                 |
+|alt              | string  | This is the alt test for your image                                           |
+|srcSet           | string  | This is an optional string you can pass in if you want to add a custom srcset |
+|className        | string  | This allows you to add custom styles to the component                         |
+|otherProps       |         | This allows you to add custom props to the component                          |
```

Comment:
I wouldn't specify `otherProps` as being a prop (as it isn't). Probably fine to omit this.